test/full-color.js

```diff
@@ -0,0 +1,65 @@
+import test from 'ava';
+import chalkTemplate from '../index.js';
+
+test('correctly parse and evaluate color-convert functions', t => {
+	t.is(chalkTemplate`{bold.rgb(144,10,178).inverse Hello, {~inverse there!}}`,
+		'\u001B[1m\u001B[38;2;144;10;178m\u001B[7mHello, '
+		+ '\u001B[27m\u001B[39m\u001B[22m\u001B[1m'
+		+ '\u001B[38;2;144;10;178mthere!\u001B[39m\u001B[22m');
+
+	t.is(chalkTemplate`{bold.bgRgb(144,10,178).inverse Hello, {~inverse there!}}`,
+		'\u001B[1m\u001B[48;2;144;10;178m\u001B[7mHello, '
+		+ '\u001B[27m\u001B[49m\u001B[22m\u001B[1m'
+		+ '\u001B[48;2;144;10;178mthere!\u001B[49m\u001B[22m');
+});
+
+test('properly handle escapes', t => {
+	t.is(chalkTemplate`{bold hello \{in brackets\}}`,
+		'\u001B[1mhello {in brackets}\u001B[22m');
+});
+
+test('throw if there is an unclosed block', t => {
+	t.throws(() => chalkTemplate`{bold this shouldn't work ever\}`, {
```

Comment:
```suggestion
	t.throws(() => {
		chalkTemplate`{bold this shouldn't work ever\}`;
	}, {
```

For readability. I prefer not using inline arrow function for void functions.

Reply:
After changing it out, it appears that `xo` doesn't like me not assigning the return value to anything

Reply:
Can you not do `void chalkTemplate`{bold this shouldn't work ever\}`;`? If not, just use an ESLint ignore comment.